package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "eslint-plugin-eunomia",
+  "version": "0.0.0",
+  "description": "Home for all codified liting and style rules for The Agency",
```

Comment:
s/liting/linting/

This statement isn't accurate if this repo is purely for eslint.

Reply:
The intent is to move the rest but this is a first step